.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+env:
+  RUSTINFRA_DEPLOY_DIR: public
+import:
+  - rust-lang/simpleinfra/travis-configs/static-websites.yml
+install:
+  - go get github.com/gohugoio/hugo
```

Comment:
I've read somewhere that this is not the recommended way of installing hugo.
Read about doing ...
```
  - curl -LO https://github.com/gohugoio/hugo/releases/download/v0.55.4/hugo_0.55.4_Linux-64bit.deb
  - sudo dpkg -i hugo_0.55.4_Linux-64bit.deb
```
But I'm not sure, I'd say let's investigate that a bit?.

Reply:
I've changed this to use a pinned version.